openmetadata-ui/src/main/resources/ui/playwright/utils/activityFeed.ts

```diff
@@ -59,3 +74,67 @@ export const deleteFeedComments = async (page: Page, feed: Locator) => {
 
   await deleteResponse;
 };
+
+export const reactOnFeed = async (page: Page) => {
+  for (const reaction of FEED_REACTIONS) {
+    await page
+      .locator(
+        '[data-testid="activity-feed-widget"] [data-testid="message-container"]:first-child'
+      )
+      .locator('[data-testid="feed-reaction-container"]')
+      .locator('[data-testid="add-reactions"]')
+      .click();
+
+    await page
+      .locator('.ant-popover-feed-reactions .ant-popover-inner-content')
+      .waitFor({ state: 'visible' });
```

Comment:
convert it to expect

Reply:
since it a minor change, will cover it with some other PR!